packages/ui-components/src/Library/index.js

```diff
@@ -151,6 +189,23 @@ const LibraryComponent = (props) => {
     );
   }, [itemTypeFilter]);
 
+  const renderItemVersionFilter = useCallback(() => {
+    if (props.itemTypes.length === 1) return null;
+    return (
+      <ItemVersionFilterContainer>
+        {distinctOptions.length > 1 ? (
+          <Dropdown
+            placeholder="Filter by cicero version"
+            style={{ width: "200px" }}
```

Comment:
The styling appears to be inconsistent for the dropdown compared to the search box below. The width should also be consistent.

Let us know if you need any help figuring it out, it's not immediately obvious to me what the issue is.